src/main/c/ssl.c

```diff
@@ -1618,6 +1753,7 @@ TCN_IMPLEMENT_CALL(jlong, SSL, getTime)(TCN_STDARGS, jlong ssl)
     if (ssl_->session == NULL) {
         // BoringSSL does not protect against a NULL session. OpenSSL
         // returns 0 if the session is NULL, so do that here.
+        // TODO(nmittler): should we throw here?
```

Comment:
@nmittler nope... we still may call this method if the session is not created yet even within netty. So I think we should not throw here otherwise it will be just harder to guard